pkg/cmd/repo/list/list.go

```diff
@@ -55,6 +55,12 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 		Use:     "list [<owner>]",
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List repositories owned by user or organization",
+		Long: heredoc.Docf(`
+                  List and filter repositories owned by a user or organization.
+
+                  When listing repositories for an organization, this will not include repositories owned by members of the organization.
+		  This also applies when using the %[1]s--archived%[1]s, %[1]s--no-archived%[1]s, %[1]s--fork%[1]s, and %[1]s--source%[1]s options.
```

Comment:
I think that the confusion here mostly lies around `--fork` and `--source`, and the rest actually hindered my understanding of the original PR. What do you think about something like:

```
Note that the list will only include repositories owned by the provided argument, and the `--fork` or `--source`
flags will not traverse ownership boundaries. For example, when listing the forks in an organization, the output
would not include those owned by individual users.
```

I think this provides the necessary clarification. If you feel this is appropriate let me know and I will make the change as there are also linting and compilation errors to resolve.

Reply:
That makes sense to me! Updating now